ui/src/components/VaultSummary.jsx

```diff
@@ -16,6 +16,7 @@ import {
   TableRow,
 } from '@material-ui/core';
 
+import { calculateCurrentDebt } from '@agoric/run-protocol/src/interest-math';
```

Comment:
we'll want to import this at the package level eventually

Reply:
yeah. not possible yet and low priority, just pointing out that we want to avoid deep imports

Reply:
Gotcha makes sense